### DISCUSSION TITLE
scss 클래스명에 줄임말을 허용할까요?

### DISCUSSION THREAD

**Discussion by uraflower** [KakaoFunding/front-end#60] [General]
### `icon`, `button`, `product` vs. `ico`, `btn`, `prod`

> 전에도 이 얘기를 했었던 것 같은데, 줄이지 말고 그냥 쓰자는 의견으로 결정되었던 것 같습니다. (맞나요? 🤔)

저는 클래스 이름이 리액트 JSX 안에서 보여지는 만큼, 간결하면서도 의미를 잘 나타냈으면 좋겠는데요.
후자처럼 줄임말을 허용하자!고 하면 어디부터 어디까지 허용할 것인지 정해야 하는 게 어려우니까,
몇몇 키워드만 정해놓고, 개발하다가 이거 줄였으면 좋겠다 싶은 건 디코로 이야기해서 컨펌받는 식으로도 편하게 작업하는 건 어떨까요?

예를 들어서 `ico`, `btn`, `prod`는 허용하자고 정해놓고, 개발하다보니 `thumbnail`이 너무 길어서 `thumb`로 줄이고 싶어지면, 디코로 물어보는 거죠.. 어떤가용?

**Comment by ohgus** [comment]
습관적으로 하다 보니 이 부분까지 생각하지 못한거 같네요...
제 생각에는 줄여도 보기에 의미가 명확한 것들은 저희끼리 정해놓고 줄여서 사용하는 것도 좋다고 생각합니다.
그리고 미림님 의견처럼 너무 길어서 줄이는 것도 서로 동의하에 하면 좋을거 같네요!

**Comment by devkyoung2** [comment]
미림님 의견에 동의합니다~
